fxa-client/src/lib.rs

```diff
@@ -496,6 +496,15 @@ impl FirefoxAccount {
         self.state.config.token_server_endpoint_url()
     }
 
+    pub fn get_connection_success_endpoint_url(&self) -> Result<Url> {
+        match self.state.config.content_url_path("connect_another_device") {
+            Ok(url) => {
+                Ok(Url::parse_with_params(&url.to_string(), &[("showSuccessMessage", "true")])?)
```

Comment:
You should use [`query_pairs_mut`](https://docs.rs/url/1.7.1/url/struct.Url.html#method.query_pairs_mut) on the returned url instead (note that this means the above would need to be `let mut`)